test/recipes/90-test_sslapi.t

```diff
@@ -41,7 +41,10 @@ unless ($no_fips) {
                  srctop_file("test", "recipes", "90-test_sslapi_data",
                              "passwd.txt"), $tmpfilename, "fips",
                  srctop_file("test", "fips-and-base.cnf"),
-                 srctop_file("test", "recipes", "90-test_sslapi_data", "dhparams.pem")])),
+                 srctop_file("test"
+                             , "recipes"
+                             , "90-test_sslapi_data"
+                             , "dhparams.pem")])),
```

Comment:
This is wrong, the `,` should be always kept with the previous argument as there is no space before it. Also there is no need to split every parameter on individual line. But that is just a matter of individual taste and not a rule so that would be ok as is.